docs/source/tools/sandbox.rst

```diff
@@ -26,23 +26,19 @@ Sandbox can also be run manually as in this example::
 
 Here, ``da run sandbox --`` tells the SDK Assistant to run ``sandbox`` from the active SDK release and pass it any arguments that follow. The example passes the DAR file to load (``Main.dar``) and the optional ``--scenario`` flag tells Sandbox to run the ``Main:example`` scenario on startup. The scenario must be fully qualified; here ``Main`` is the module and ``example`` is the name of the scenario, separated by a ``:``. The scenario is used for testing and development; it is not run in production.
 
+
+Running with persistence
+************************
+
+By default the Sandbox uses an in-memory store and as such it loses its state when stopped or restarted. There is a possibility to use a Postgres database for persistence, which allows the Sandbox to be shut down and continued later where it left off.
+
+The application has to have access to an initially empty database, and the database user needs to have authority to execute DDL operations as the Sandbox manages its own database schema, applying migrations if necessary when upgrading versions. 
```

Comment:
```suggestion
To set this up, you must:
- create an initially empty Postgres database that the Sandbox application can access 
- have a database user for Sandbox that has authority to execute DDL operations 

  This is because the Sandbox manages its own database schema, applying migrations if necessary when upgrading versions. 
```